.github/workflows/botonic-plugin-contentful-tests.yml

```diff
@@ -3,11 +3,8 @@ name: Botonic plugin-contentful tests
 on:
   push:
     paths:
-      - '*'
-      - '.github/**'
-      - 'packages/*'
       - 'packages/botonic-plugin-contentful/**'
```

Comment:
please restore '*'' and 'packages/*' There are there build configurations

Reply:
ok, sorry, I thought they were not needed. I pushed a new commit with the changes